### PR TITLE
Dev unasl crit tool fixes

### PR DESCRIPTION
**Pull request by compor**
This PR does the following:

- Minor fixes to support Python 3.
- Converts absolute imports of `pycriu` Python library to relative.
- Allows configuring the name of the `pycriu` Python library to allow multiple installations and correct imports in the `crit` commandline tool.
- Fixes build generation of the `crit` commandline tool in order to be always up to date with the last `make` invocation.
- Enhances documentation with the above information.

This addresses https://github.com/systems-nuts/UnASL/issues/165

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -49,6 +49,16 @@ If a package was installed after a failed compilation, use "make mrproper-top" t
 
 Additional packages maybe needed, please refer to: https://criu.org/Installation
 
+In order to allow multiple installations to work in conjunction with one another (i.e., no conflicts or shadow imports)
+the environment variable `PY_PKG_DIR` which defaults to "pycriu" (i.e., `import pycriu`).
```
Comment (blackgeorge-boom) [inline INSTALL.md]: Here you mean that we must set `PY_PKG_DIR`?

Comment (compor) [reply INSTALL.md]: oh, something is truncated here, Let me amend it.

Comment (compor) [reply INSTALL.md]: Updated.

**Review by blackgeorge-boom — APPROVED** [review]
The changes seem straightforward, thanks a lot for this! LGTM.

**Pull request merged**